### DISCUSSION TITLE
readDir with possiblity to read directories

### DISCUSSION THREAD

**Discussion by vasicvuk** [helmfile/helmfile#253] [Ideas]
I wanted to use the `readDir` function within the template to get all subdirectories within the folder. I would then use `isFile` function to check if the string is a directory. The current implementation supports only listing files and not directories, so this is impossible.

My suggestion is to implement a function named `readDirs` which would then list only directories instead of files to enable users to also work with multi-level folders.

I would like to implement this as PR if possible.

**Comment by vasicvuk** [comment]
Pull request with implementation is here:

https://github.com/helmfile/helmfile/pull/254

**Comment by yxxhero** [comment]
I write comments in PR. Thanks very much.

**Reply by yxxhero** [reply]
done

**Comment by vasicvuk** [comment]
Great, thanks for collaboration on this feature :)

**Comment by yxxhero** [comment]
@vasicvuk Thanks very much.
